src/models/resources/actors_decode.js

```diff
@@ -0,0 +1,18 @@
+const heroTokensBad = {
+  "*Hero009*"   : "Krul",
+  "*Hero010*"   : "Skaarf",
+  "*Sayoc*"     : "Taka",
+  "*Hero016*"   : "Rona",
+};
+
+export function decodeActorName(token) {
+  // Returns the hero name, given a token.
+  //
+  // For "*Hero009*", etc., returns "Krul", using heroTokensBad look-up table.
+
+  let match = /^\*(.*)\*$/.exec(token);
+  if (match) return (heroTokensBad[token] || match[1]);
+
+  // If somehow it doesn't match, let's just return the token
+  return token;
```

Comment:
This is fine, but keeping with the standard set with heroTokensBad, we'd probably want to remove all instances of the asterisk. 

```js
return token.replace(/\*/g, '')
```